gradle.properties

```diff
@@ -0,0 +1,13 @@
+groupId=info.novatec
```

Comment:
Group-ID would not be Novatec - I personally we don't need a Group-ID and version at all.
(unless Quarkus is in some way different to Spring Boot in that regard)

But if you absolutely want one it should be `io.github.test-automation-in-practice`.